docs/user/translating.rst

```diff
@@ -164,9 +164,9 @@ The following keyboard shortcuts can be utilized during translation:
 :kbd:`Ctrl+<NUMBER>` or :kbd:`Cmd+<NUMBER>`
     Copies placeable of given number from source string.
 :kbd:`Ctrl+M <NUMBER>` or :kbd:`Cmd+M <NUMBER>`
-    Copy machine translation of given number to current translation.
+    Copy the machine translation of given number to current translation.
 :kbd:`Ctrl+I <NUMBER>` or :kbd:`Cmd+I <NUMBER>`
-    Ignore failing check of given number.
+    Ignore failing checks beyond a given number.
```

Comment:
I don't think beyond is correct here- `Ctrl+I 1` dismisses first failing check, `Ctrl+I 2` dismisses second failing check and so on.